src/components/event-log/event-log.scss

```diff
@@ -0,0 +1,153 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/font.yml';
+@use '../../design-tokens/opacity.yml';
+@use '../../design-tokens/sizes.yml';
+@use '../../mixins/theme';
+
+/// The breakpoint at which items should attempt to display in a single line.
+/// @access private
+$_breakpoint-wide: breakpoint.$m;
+
+/**
+ * Themed properties
+ */
+
+@include theme.props() {
+  --theme-color-border-event-log-item: #{colors.$gray-light};
+  --theme-opacity-event-log-location: #{opacity.$muted};
+}
+
+@include theme.props(dark) {
+  --theme-color-border-event-log-item: #{colors.$primary-brand-dark};
+  --theme-opacity-event-log-location: 1;
+}
+
+/**
+ * Main container
+ *
+ * 1. Defining content columns here, which will be referenced by subgrid in the
+ *    items themselves. This will keep the date column nicely sized in supported
+ *    browsers instead of relying on an arbitrary list.
+ */
+
+.c-event-log {
+  display: grid;
+  grid-column-gap: sizes.$gap-inline-medium;
+  grid-template-columns: auto 1fr; /* 1 */
+}
+
+/**
+ * Individual item container
+ *
+ * 1. Spans the full width of the container.
+ * 2. In browsers that don't support subgrid, we manually define columns. The
+ *    first column width is a "magic number" that looks okay most of the time
+ *    for a date in the format of "Mon D, YYYY".
+ */
+
+.c-event-log__item {
+  display: grid;
```

Comment:
Do we need any fallback for non-grid browsers? Or are we fine with default styles at that point?

Reply:
I was unable to actually test this in a browser without Grid since [storybook fails to load in IE11](https://github.com/storybookjs/storybook/issues/8884), however I tried disabling the grid rule and it looked fine, so I think we should be fine here: 

![Uploading Screen Shot 2020-12-03 at 4.34.41 PM.png…]()